pdr_backend/lake/etl.py

```diff
@@ -241,78 +302,31 @@ def _calc_bronze_start_end_ts(self) -> Tuple[UnixTimeMs, UnixTimeMs]:
 
         return from_timestamp, to_timestamp
 
-    @enforce_types
-    def create_etl_view(self, table_name: str):
-        # Assemble view query and create the view
-        assert (
-            table_name in _ETL_REGISTERED_TABLE_NAMES
-        ), f"{table_name} must be a bronze table"
-
-        db = DuckDBDataStore(self.ppss.lake_ss.lake_dir)
-        temp_table = TempTable(table_name)
-        etl_view = ETLTable(table_name)
-
-        table_exists = db.table_exists(table_name)
-        temp_table_exists = db.table_exists(temp_table.fullname)
-        etl_view_exists = db.view_exists(etl_view.fullname)
-        assert temp_table_exists, f"{temp_table.fullname} must already exist"
-        if etl_view_exists:
-            logger.error("%s must not exist", etl_view.fullname)
-            return
-
-        view_query = None
-
-        if table_exists and temp_table_exists:
-            view_query = """
-                CREATE VIEW {} AS
-                (
-                    SELECT * FROM {}
-                    UNION ALL
-                    SELECT * FROM {}
-                )""".format(
-                etl_view.fullname,
-                table_name,
-                temp_table.fullname,
-            )
-            db.query_data(view_query)
-            logger.info(
-                "  Created %s view using %s table and %s temp table",
-                etl_view.fullname,
-                table_name,
-                temp_table.fullname,
-            )
-        else:
-            view_query = f"CREATE VIEW {etl_view.fullname} AS SELECT * FROM {temp_table.fullname}"
-            db.query_data(view_query)
-            logger.info(
-                "  Created %s view using %s temp table",
-                etl_view.fullname,
-                temp_table.fullname,
-            )
-
-    def update_bronze_pdr(self):
+    def do_bronze_queries(self):
         """
         @description
-            Update bronze tables
+            Run all bronze queries to process new + update events
         """
-        logger.info("update_bronze_pdr - Update bronze tables.")
+        logger.info("do_bronze_queries - start")
 
         # st_timestamp and fin_timestamp should be valid UnixTimeMS
         st_timestamp, fin_timestamp = self._calc_bronze_start_end_ts()
+        db = DuckDBDataStore(self.ppss.lake_ss.lake_dir)
 
-        for dataclass in _ETL_REGISTERED_LAKE_TABLES:
-            etl_func = dataclass.get_fetch_function()
-            etl_func(
-                path=self.ppss.lake_ss.lake_dir,
-                st_ms=st_timestamp,
-                fin_ms=fin_timestamp,
+        # if st_ts or fin_ts != ppss, then we must have records somewhere
+        is_first_run: bool = self._clamp_checkpoints_to_ppss is True or (
```

Comment:
if _clamp_checkpoints_to_ppss is typed to boolean, there is no need for "is True"